contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -0,0 +1,600 @@
+#include <sys/zfs_context.h>
```

Comment:
Please add a license header to this new file.  I'd recommend the [Modified BSD License](https://en.wikipedia.org/wiki/BSD_licenses#3-clause) which is what the [Linux PAM](https://en.wikipedia.org/wiki/Linux_PAM) is dual licensed under.

Reply:
Is the license header good like this?

Reply:
Thanks, that'll work.  Please just move the `Copyright...` lines after the license text.